Cannot compile due to more strict zcert interface

Apparently, the new generated C interface of zcert is more strict. When compiling CZMQ, I get:

```
make
Making all in doc
make[1]: Nothing to be done for `all'.
  CC       src/src_libczmq_la-zactor.lo
  CC       src/src_libczmq_la-zauth.lo
  CC       src/src_libczmq_la-zarmour.lo
  CC       src/src_libczmq_la-zbeacon.lo
  CC       src/src_libczmq_la-zcert.lo
src/zcert.c:240:60: error: format string is not a string literal (potentially insecure) [-Werror,-Wformat-security]
                zcert_set_meta (self, zconfig_name (item), zconfig_value (item));
                                                           ^~~~~~~~~~~~~~~~~~~~
1 error generated.
make[1]: *** [src/src_libczmq_la-zcert.lo] Error 1
make: *** [all-recursive] Error 1
```

[Here](https://github.com/zeromq/czmq/commit/085deec971eaeb952cf13fa2d6509d98dc1571ad#diff-6270c37246eefa96a34041ff22c798e4R58) is the relevant line in the commit that changed the interface.

And [here](https://github.com/zeromq/czmq/blob/master/src/zcert.c#L240)'s the line that tries to use the function.

I have no idea how to fix this.

--- a/src/zcert.c
+++ b/src/zcert.c
@@ -237,7 +237,7 @@ zcert_load (const char *filename)
             zconfig_t *metadata = zconfig_locate (root, "/metadata");
             zconfig_t *item = metadata? zconfig_child (metadata): NULL;
             while (item) {
-                zcert_set_meta (self, zconfig_name (item), zconfig_value (item));
+                zcert_set_meta (self, zconfig_name (item), "%s", zconfig_value (item));
                 item = zconfig_next (item);
             }
         }

--- a/src/zmakecert.c
+++ b/src/zmakecert.c
@@ -30,7 +30,7 @@ s_get_meta (zcert_t *cert, char *prompt, char *name)
     if (strlen (value) && value [strlen (value) - 1] == '\n')
         value [strlen (value) - 1] = 0;
     if (*value)
-        zcert_set_meta (cert, name, value);
+        zcert_set_meta (cert, name, "%s", value);
     return 0;
 }
 
@@ -46,7 +46,7 @@ int main (void)
         
     char *timestr = zclock_timestr ();
     zcert_set_meta (cert, "created-by", "CZMQ zmakecert");
-    zcert_set_meta (cert, "date-created", timestr);
+    zcert_set_meta (cert, "date-created", "%s", timestr);
     free (timestr);
     zcert_dump (cert);
     zcert_save (cert, "mycert.txt");

